pkg/kf/apps/log_tailer.go

```diff
@@ -114,24 +114,36 @@ func (t *pushLogTailer) handleWatch() (bool, error) {
 	defer ws.Stop()
 
 	for e := range ws.ResultChan() {
-		app, ok := e.Object.(*v1alpha1.App)
-		if !ok {
+		switch e.Object.(type) {
+		case *v1alpha1.App:
+			app := e.Object.(*v1alpha1.App)
+
```

Comment:
I proposed a fix in #793 I think could go here, WDYT?

Reply:
Love it!